stats.go

```diff
@@ -214,7 +214,10 @@ type StatGenerator interface {
 // NewStore returns an Empty store that flushes to Sink passed as an argument.
 // Note: the export argument is unused.
 func NewStore(sink Sink, _ bool) Store {
-	return &statStore{sink: sink}
+	return &statStore{
+		sink: sink,
+		conf: GetSettings(), // todo: right now the environment is being loaded in multiple places and can be made more efficient
```

Comment:
The [sync.OnceValue](https://pkg.go.dev/sync#OnceValue) function can help eliminate these multiple loads.